packages/core/src/components/select/select.tsx

```diff
@@ -71,8 +71,14 @@ export class AtomSelect {
       const optionElement =
         this.getElementByTag(itemElement, 'ion-radio') ||
         this.getElementByTag(itemElement, 'ion-checkbox')
+
+      if (!optionElement?.shadowRoot) return
```

Comment:
were you able to reproduce it?

Reply:
I couldn't reproduce the error, so I decided to make this change and go with the production testing approach. LOL.

Reply:
That's ok. Did you create another task to figure out what's going on? then we can merge this first PR and explore the root case into another.

Reply:
created here https://juntossomosmais.monday.com/boards/3345740155/views/76424285/pulses/7675362982